.storybook/utils.js

```diff
@@ -0,0 +1,78 @@
+const path = require('path');
+
+const matchRelativePath = /^\.\.?[/\\]/;
+
+function isAbsolutePath(str) {
+  return path.posix.isAbsolute(str) || path.win32.isAbsolute(str);
+}
+
+function isRelativePath(str) {
+  return matchRelativePath.test(str);
+}
+
+// TODO simplify for the next major release
```

Comment:
We will need to remember to link this to a ticket :) 

Although would wonder why it needs to be ready for the next major as this is just in the storybook folder so never reaches a consumer anyway